src/pages/AppDetails/AppDetailsPage.tsx

```diff
@@ -56,7 +56,7 @@ class AppDetails extends React.Component<RouteComponentProps<AppId>, AppDetailsS
         });
       })
       .catch(error => {
-        MessageCenter.add(API.getErrorMsg('Could not fetch App Details.', error));
+        MessageCenter.add(API.getErrorMsg('Could not fetch App Details', error));
```

Comment:
I just want to point out that patternfly seems to end sentences with "." for [toasts](https://www.patternfly.org/pattern-library/communication/toast-notifications/) and [notifications](https://www.patternfly.org/pattern-library/communication/notification-drawer/)

Reply:
yes but the text here is a part of a bigger message, and for that reason punctuation should be avoided. IIRC it's followed by a coma.

Reply:
Oh ok, thanks for the clarification :)